charts/ingress-nginx/Chart.yaml

```diff
@@ -3,6 +3,7 @@ annotations:
      - "helm: Fix opentelemetry module installation for daemonset (#9792)"
      - "Update charts/* to keep project name display aligned (#9931)"
      - "Update Ingress-Nginx version controller-v1.8.0"
+     - "Add loadBalancerClass params on service (#9562)"
```

Comment:
this is not needed, we will add it to the release notes.

Reply:
I removed it in last push